### PR TITLE
Release 12.0.0 (#2966)

### PR DESCRIPTION
**Pull request by npauzenga**
* bump cumulus version to 12.0.0

* bump website documentation to 12.0.0

**Summary:** Summary of changes

Addresses [CUMULUS-XX: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-XXX)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [ ] Update CHANGELOG
- [ ] Unit tests
- [ ] Ad-hoc testing - Deploy changes and test manually
- [ ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## Unreleased
```
Comment (jennyhliu) [inline CHANGELOG.md:7]: Please update the `## Unreleased`, there are a few merges after the release `v12.0.0` was cut

File: CHANGELOG.md [hunk 2]
```diff
@@ -5934,7 +5936,8 @@ Note: There was an issue publishing 1.12.0. Upgrade to 1.12.1.
 
 ## [v1.0.0] - 2018-02-23
 
-[unreleased]: https://github.com/nasa/cumulus/compare/v11.1.1...HEAD
+[unreleased]: https://github.com/nasa/cumulus/compare/v12.0.0...HEAD
+[v12.0.0]: https://github.com/nasa/cumulus/compare/v11.1.0...v12.0.0
```
Comment (jennyhliu) [inline CHANGELOG.md:5940]:
```suggestion
[v12.0.0]: https://github.com/nasa/cumulus/compare/v11.1.1...v12.0.0
```

File: CHANGELOG.md (hunk 1 above)
Comment (npauzenga) [reply CHANGELOG.md]: Thanks @jennyhliu! I've updated the changelog to match the entries from the 12.0.0 release: https://github.com/nasa/cumulus/pull/2966/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4ed

Everything else is moved to "Unreleased".

https://github.com/nasa/cumulus/pull/2971/commits/b25c258430661d611c398055c0b5d2c4f555b9c4

Comment (npauzenga) [reply CHANGELOG.md]: And just as I did that there was another merge to master 😄 https://github.com/nasa/cumulus/pull/2971/commits/b88db6134ecc6cabf5b32eeadb227bf328a78f1f

**Review by jennyhliu — APPROVED** [review]
👍

**Pull request merged**